migrations/schemas/20250320153030-create-icy_btc_swap_request.sql

```diff
@@ -0,0 +1,19 @@
+-- +migrate Up
+CREATE TABLE icy_swap_btc_requests (
+   id uuid PRIMARY KEY DEFAULT (uuid()),
+   profile_id varchar(255) NOT NULL,
+   request_code varchar(255) NOT NULL,
+   tx_status varchar(255),
+   tx_id int,
+   btc_address varchar(255),
+   timestamp integer,
+   amount varchar(255),
+   token_name varchar(255),
+   token_id varchar(255),
+   swap_request_status varchar(255),
+   revert_status varchar(255),
```

Comment:
Consider adding indexes for frequently queried columns such as `profile_id` and `request_code` to improve query performance:
```sql
CREATE INDEX idx_icy_swap_btc_profile_id ON icy_swap_btc_requests(profile_id);
CREATE INDEX idx_icy_swap_btc_request_code ON icy_swap_btc_requests(request_code);
```